README.md

```diff
@@ -100,12 +100,12 @@ jobs:
         uses: neofinancial/ticket-check-action@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          ticketLink: 'https://app.clubhouse.io/:org/story/%ticketNumber%'
-          ticketPrefix: 'CH-'
-          titleRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          branchRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyRegex: '(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyURLRegex: 'http(s?):\/\/(app.clubhouse.io)(\/:org)(\/story)\/(?<ticketNumber>\d+)'
+          ticketLink: 'https://app.shortcut.com/:org/story/%ticketNumber%'
+          ticketPrefix: 'SC-'
+          titleRegex: '^(CH|sc)(-?)(?<ticketNumber>\d+):'
+          branchRegex: '^(CH|sc)(-?)(?<ticketNumber>\d+):'
+          bodyRegex: '(CH|sc)(-?)(?<ticketNumber>\d+):'
+          bodyURLRegex: 'https?:\/\/app\.(clubhouse.io|shortcut.com)(\/:org)\/story\/(?<ticketNumber>\d+)\/.+'
```

Comment:
```suggestion
          bodyURLRegex: 'https?:\/\/app\.(clubhouse.io|shortcut.com)(\/:org)\/story\/(?<ticketNumber>\d+)'
```
We don't need to enforce that the link is `/1234/do-something` rather than `/1234`. (Both are correct links to ticket 1234)